### ISSUE TITLE
Beignet creates tx with fee bellow Minimal if amout is small

### ISSUE THREAD

**Issue by limpbrains** [synonymdev/beignet#73]
When the fees are 4,4,4 and you are trying to create tx with small amount (~700 sats) beignet will reduce tx fee and set it bellow 4. It also do not allow to rase it, because of "Error: Unable to increase the fee any further. Otherwise, it will exceed half the current sending/output amount." rule.

I think this rule should be ignored, beignet should not create tx with such a small fee, it will not be broadcasted

**Comment by pwltr** [comment]
Agreed, this rule has been causing issues. If we want to keep the warning maybe we can have a flag (and string) in the result
of the fee setting method. Then the UI can decide what to do with the warning.

Also see https://github.com/synonymdev/bitkit/issues/1704

**Comment by limpbrains** [comment]
Maybe only use this rule if the amount sent is greater than 10000 sats or something?
